src/main/webapp/app/exercises/text/assess/textblock-feedback-editor/textblock-feedback-editor.component.scss

```diff
@@ -39,3 +39,7 @@ label {
     margin-top: -0.5vh;
     float: right;
 }
+
+.impactWarningLabel {
+    //font-size: small;
```

Comment:
This looks like dead code which should be removed?

Reply:
Yes, it will be uncommented actually. Left it as reference for the testing session to get some feedback from devs [here](https://github.com/ls1intum/Artemis/pull/3581#issuecomment-863311276)